### DISCUSSION TITLE
Cannot add ID photo for application

### DISCUSSION THREAD

**Discussion by LimePencil** [community/community#123304] [GitHub Education]
### Hi everyone,

I cannot add a photo of my ID card to the application form. Every time I upload it and press "process my application", it keeps going back to the same page no matter how many times I try.

**Comment by Rishit-katiyar** [comment]
Ensure your ID photo is in supported format and size, check internet connection, try a different browser, clear cache and cookies, or contact Educational support of your institute for further assistance

**Reply by KRalfTech** [reply]
I'm having the same issue. Surely there's a github support service for this as it appears to be an issue with the website?

Edit: I have gone through all of those other steps too, by the way.

**Reply by honzbur** [reply]
Hey, same here. I uploaded a confirmation document but it doesn't work for me via taking a photo which is for some reason required. Any help please?

**Reply by Rishit-katiyar** [reply]
hi @KRalfTech and @honzbur , what is the website that you guys are using  for the image upload . maybe then i could help you better

**Reply by honzbur** [reply]
https://education.github.com/discount_requests/application

It's the next one after clicking "Continue" (there's specific number insted of Xs)

https://education.github.com/discount_requests/XXXXXXXX/additional_information

**Reply by KRalfTech** [reply]
Yea, same - https://education.github.com/discount_requests/XXXXXXXX/additional_information

**Reply by KRalfTech** [reply]
@Rishit-katiyar Thank you

**Reply by Rishit-katiyar** [reply]
@honzbur and @KRalfTech , I quickly downloaded the GitHub mobile app and then uploaded from there , it worked for me . it showed me a "thanks for uploading your photo" output. is this the problem you were having?

![IMG_20240511_183326.jpg](https://github.com/community/community/assets/167756997/1a243b07-661c-4c7e-9ae1-c6e81a903cfe)

**Reply by honzbur** [reply]
@Rishit-katiyar I could only open the link via mobile web browser and then the only available option was front camera so the photo proof is of bad quality but it went through. The proof is sent out, hope it's gonna work. Thank you for you help :)

**Reply by Rishit-katiyar** [reply]
@honzbur Lessss gooooooo , and @LimePencil @KRalfTech  did it work for you?

**Reply by KRalfTech** [reply]
It did the same. Went through to browser from the app rather than be able to do it in-app, would only come up with front facing camera and so it obviously ended up too blurry to read, I tried to retake it and it seems to not want to load at all! To be honest, I'm not going to cry about it though. Trying to navigate this site is such a pain in the *** that I'm happy to just not use it anymore. Thank you for your help though.
